src/components/donation/Steps/DonateMethods/Stripe/Frequency.tsx

```diff
@@ -28,11 +28,14 @@ export default function Frequency() {
         Frequency <span className="text-red">*</span>
       </RG.Label>
       <div className="grid grid-cols-2 gap-2 @[21rem]/frequency:flex">
-        <RG.Option value={"monthly" satisfies Freq} className={styles.option}>
+        <RG.Option
+          value={"subscription" satisfies Freq}
+          className={styles.option}
+        >
           <span>Give Monthly</span>
           <Icon type="Check" className={styles.icon} />
         </RG.Option>
-        <RG.Option value={"once" satisfies Freq} className={styles.option}>
+        <RG.Option value={"one-time" satisfies Freq} className={styles.option}>
```

Comment:
Aligned frequency union type names with AWS union type names - no need for manual conversion.